client/src/components/pages/Game.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react'
+import GameContainer from './GameContainer';
+import firstFifteenJson from "../../sampleresponse/firstfifteen.json"
+
+const Game = () => {
+
+    const [pokemonList, setPokemonList] = useState([]);
+
+    //the following is for the api call:
+    // const getPokemonList = () => {
+    //     console.log("calling api")
+    //     fetch("https://pokeapi.co/api/v2/pokemon?limit=151")
+    //         .then((response) => response.json())
+    //         .then((pokemonListResponse) => {
+    //             console.log("calling api response", pokemonListResponse)
+    //             shuffledPokemonList(pokemonListResponse.results);
+    //         });
+    // }
+
+    const getPokemonList = () => {
+        shuffledPokemonList(firstFifteenJson.results);
+    }
+
+    const shuffledPokemonList = (pokemonListParam) => {
+        const shuffled = pokemonListParam
+            .map(value => ({ value, sort: Math.random() }))
+            .sort((a, b) => a.sort - b.sort)
+            .map(({ value }) => value);
+        setPokemonList(shuffled);
+        console.log("line34", pokemonList);
+
+    }
+
+
+    useEffect(() => {
+        getPokemonList();
+        console.log("line40", pokemonList);
```

Comment:
Clean up this and any other unnecessary `console.log` statements before you merge this into main.

Reply:
Good catch, will clean up the console logs. If I intend to use the code for the api call later should I put it as a TODO or should I save the code somewhere else and delete it here?

Reply:
disregard that, was referring to the wrong lines when looking at you comment